pkg/config/store/yaml/store.go

```diff
@@ -31,6 +31,12 @@ import (
 const (
 	// DefaultPath is the default path for the configuration file
 	DefaultPath = "PROJECT"
+
+	// Comment for 'PROJECT' config file
+	commentStr = `# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
```

Comment:
That is great. Could we just add? 
```suggestion
# and allow the plugins properly work. 
# More info: https://book.kubebuilder.io/reference/project-config.html
```

So that user have more info. also, we would like to improve this doc to have a better explanation, see: https://github.com/kubernetes-sigs/kubebuilder/issues/3088

Reply:
sounds good! I've added the link to the comment